changelogs/fragments/10005-fix-method-exists-in-sysrc.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - sysrc - always reports ``changed=true`` when ``state=absent``. This fixes the method ``exists()`` (https://github.com/ansible-collections/community.general/issues/10004, https://github.com/ansible-collections/community.general/pull/10005).
```

I think with this PR it is no longer always reporting `changed=true`?
```suggestion
  - sysrc - no longer always reporting ``changed=true`` when ``state=absent``. This fixes the method ``exists()`` (https://github.com/ansible-collections/community.general/issues/10004, https://github.com/ansible-collections/community.general/pull/10005).
```